arkane/encorr/bac.py

```diff
@@ -453,6 +453,40 @@ def fit(self, db_names: Union[str, List[str]] = 'main', **kwargs):
         logging.info(f'RMSE/MAE before fitting: {stats_before.rmse:.2f}/{stats_before.mae:.2f} kcal/mol')
         logging.info(f'RMSE/MAE after fitting: {stats_after.rmse:.2f}/{stats_after.mae:.2f} kcal/mol')
 
+    def test(self,
+             species: List[ReferenceSpecies] = None,
+             dataset: BACDataset = None,
+             db_names: Union[str, List[str]] = None) -> BACDataset:
+        """
```

Comment:
The BAC class API is much more flexible than just for use in standard Arkane jobs, so I made this method so that it is easy to test fitted BACs on a variety of datasets.

Reply:
Fair enough, so long as it is intended to be used other than just for our own unit tests